types/ember/index.d.ts

```diff
@@ -2032,23 +2032,119 @@ export namespace Ember {
     function reset(): void;
     function rewatch(obj: any): void;
 
-    type RunMethod<T> = (...args: any[]) => T;
+    type RunMethod<Target, Ret = any> = ((this: Target, ...args: any[]) => Ret) | keyof Target;
+    type EmberRunQueues = "sync" | "actions" | "routerTransitions" | "render"| "afterRender" | "destroy";
+
+    interface EmberRunTimer {
+        __ember_run_timer_brand__: any;
```

Comment:
One thought — maybe a `never` on these "brands", just in case someone goofily sees it and tries to use it? Does that work correctly?

Reply:
I think that would work. I noticed typescript's own typings do it this way so I didn't think more about it